Find a functionally equivalent code:android.view.ViewGroup.resetTouchState:COMMENT
Method Modifier: private     
Comment:/**
 * Resets all touch state in preparation for a new cycle.
 */

@@ -2,5 +2,6 @@
     clearTouchTargets();
     resetCancelNextUpFlag(this);
     mGroupFlags &= ~FLAG_DISALLOW_INTERCEPT;
+    mNestedScrollAxes = SCROLL_AXIS_NONE;
 }

